src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let price = days * 40;
```

Comment:
Prefer `const` over `let` where possible, to avoid unintentional changes. Here `price` can be a `const` because it's not reassigned in the `if` conditions, it's just being modified.